cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -964,6 +980,10 @@ def _validate_uns_dict(self, uns_dict: dict) -> None:
             elif value is not None and not isinstance(value, (np.bool_, bool, numbers.Number)) and len(value) == 0:
                 self.errors.append(f"uns['{key}'] cannot be an empty value.")
 
+            value_def = dict_def["keys"].get(key, None)
```

Comment:
i had to update `_validate_uns_dict` to support validating against curie constraints